docs/install/README.md

```diff
@@ -392,29 +392,43 @@ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3
 ### Install K8ssandra Operator
 The GitHub Actions for the project are configured to build and push a new operator image to Docker Hub whenever commits are pushed to `main`. 
 
-See [here](https://hub.docker.com/repository/docker/k8ssandra/k8ssandra-operator/tags?page=1&ordering=last_updated) on Docker Hub for a list of availabe images.
+See [here](https://hub.docker.com/repository/docker/k8ssandra/k8ssandra-operator/tags?page=1&ordering=last_updated) on Docker Hub for a list of available images.
 
-Create a kustomization directory that builds from the `main` branch:
+If you wish to install the default installation, one can install with kubectl:
+
+```console
+kubectl apply -k github.com/k8ssandra/k8ssandra-operator/config/deployments/default
+```
+
+In case you want to customize the installation, create a kustomization directory that builds from the `main` branch and in this case we'll add namespace creation and define new namespace. Note the `namespace` property which we added. This property tells Kustomize to apply a transformation on all resources that specify a namespace.
 
 ```sh
 K8SSANDRA_OPERATOR_HOME=$(mktemp -d)
 cat <<EOF >$K8SSANDRA_OPERATOR_HOME/kustomization.yaml
+
+namespace: k8ssandra-operator
+
 resources:
-- github.com/k8ssandra/k8ssandra-operator/config/default?ref=main
+- github.com/k8ssandra/k8ssandra-operator/config/deployments/default?ref=main
+
+components:
+- github.com/k8ssandra/k8ssandra-operator/config/components/namespace
 
 images:
 - name: k8ssandra/k8ssandra-operator
   newTag: 7a2d65bb
 EOF
 ```
+
 Now install the operator:
 
 ```
 kustomize build $K8SSANDRA_OPERATOR_HOME | kubectl apply -f -
```

Comment:
Maybe change this to `kubectl apply -k $KSSANDRA_OPERATOR_HOME` to be consistent.